src/context/authStore/bucketStorageHelper.test.ts

```diff
@@ -0,0 +1,279 @@
+import * as SecureStore from "expo-secure-store";
+import {
+  deleteInStoreInBuckets,
+  readFromStoreInBuckets,
+  saveToStoreInBuckets,
+} from "./bucketStorageHelper";
+
+jest.mock("expo-secure-store");
+const mockedSecureStore = SecureStore as jest.Mocked<typeof SecureStore>;
+const mockSecureGetItem = mockedSecureStore.getItemAsync;
+const mockSecureSetItem = mockedSecureStore.setItemAsync;
+const mockSecureDeleteItem = mockedSecureStore.deleteItemAsync;
+
+const testKey = "testkey";
+
+const veryLongStringA = "A".repeat(1000);
+const veryLongStringB = "B".repeat(1000);
+const veryLongStringC = "C".repeat(1000);
+
+describe("bucket storage helpers", () => {
+  beforeEach(() => {
+    mockSecureGetItem.mockReset();
+    mockSecureSetItem.mockReset();
+    mockSecureDeleteItem.mockReset();
+  });
+
+  describe("saveToStoreInBuckets", () => {
+    it("should split up data into buckets according to the specified size", async () => {
+      expect.assertions(4);
+
+      await saveToStoreInBuckets(
+        testKey,
+        JSON.stringify({
```

Comment:
Can we move this out to a constant, noticed it's used in the following expect statements